### PR TITLE
feat: custom metadata for imports

### PR DESCRIPTION
**Pull request by antfu**

### CODE REVIEW

**Review by danielroe — APPROVED** [review]
This looks good to me. 👌

File: src/types.ts [hunk 1]
```diff
@@ -15,6 +15,17 @@ export interface ImportCommon {
...
   /** If this import is disabled */
   disabled?: boolean
+  /**
+   * Metadata of the import
+   */
+  meta?: {
+    /** Short description of the import */
+    description?: string
+    /** URL to the documentation */
+    docsUrl?: string
```
Comment (danielroe) [inline src/types.ts:25]: would we want metadata for presets (e.g. single docs url for entire bundle that gets inherited by individual imports?)

**Comment by pi0** [comment]
Nice addition. Specially this can later be helpful to support virtual auto imports too! Would this later override TSDocs for auto imports or what is usage for meta?

File: src/types.ts (hunk 1 above)
Comment (antfu) [reply src/types.ts:25]: Actually it's already inherited :)

Comment (danielroe) [reply src/types.ts:25]: Perfect!

**Comment by antfu** [comment]
Being able to use TSDocs would be great!

> what is usage for meta?

We could be able to provide some description and docs view in the devtools.

**Pull request merged**